helpers/src/normalizers.js

```diff
@@ -23,6 +23,7 @@ export const doublePercentDecoder = decoderMaker(
 export function gatedEncoder(encoder, ...flagNames) {
     const fn = function gatedEncoder(str, flags = {}) {
         const run = flagNames.reduce((run, flag) => run || (flag.charAt(0) === '!' ? !flags[flag.substring(1)] : flags[flag]), false);
+        str = typeof str === 'string' ? str : str.v;
```

Comment:
this prevents the `encoder()` to receive the full object. i think this is only needed when `run` is false.

Reply:
Changed as suggested